pipestat_example/pipeline/pipestat_output_schema.yaml

```diff
@@ -0,0 +1,28 @@
+pipeline_name: test_pipe
+samples:
```

Comment:
this is not the correct output schema. write an actual output schema for this pipeline.